include/abb_librws/common/rw/panel.h

```diff
@@ -73,6 +73,25 @@ namespace abb :: rws :: rw
 
     std::ostream& operator<<(std::ostream& os, OperationMode mode);
 
+    /**
+     * @brief Set robot operation mode.
+     *
+     * The documentation strings are taken from 
+     * https://developercenter.robotstudio.com/api/RWS?urls.primaryName=Panel%20Service -> POST rw/panel/mode
+     */
+    enum class SetOperationMode
+    {
+        // State manual mode
+        man,
+        // State manual mode & full speed
+        manF,
+        // State automatic mode
+        automatic,
+    };
+
+
+    std::ostream& operator<<(std::ostream& os, SetOperationMode mode);
+
```

Comment:
This is not a good idea to have two separate types representing the same thing. "Controller operation modes" is one thing. Imagine you have a code:
```c++
void foo(RWSClient& client, OperationMode mode)
{
    if (getOperationMode(client) != mode)
    {
        do_something();
        setOperationMode(client, mode);
    }
}
```
what would you do if `getOperationMode()` and `setOperationMode()` used different types?

Not every `OperationMode` is a valid argument value for `setOperationMode()`, but this is perfectly fine -- you can throw `std::invalid_argument`.